unittests/wsdl_document/test_wsdl_document.cpp

```diff
@@ -1060,7 +1060,7 @@ void WsdlDocumentTest::testSendTelegram()
 
     // Check the request as received by the server.
     const QByteArray expectedRequestXml =
-        QByteArray(xmlBegin) + "<TelegramRequest " + xmlNamespaces + ">"
+        QByteArray(xmlBegin) + "<TelegramRequest " + xmlNamespaces + " xmlns:n1=\"http://www.kdab.com/xml/MyWsdl/\">"
```

Comment:
(what's strange with this one is that n1 isn't actually used... doesn't hurt though, I guess)